here/here_readme.md

```diff
@@ -0,0 +1,218 @@
+![image](https://user-images.githubusercontent.com/46324452/149184544-bbff447b-bba7-4585-aebf-d9fc65d21998.png)
+
+This is the coverage of the congestion network. 
+![image](https://user-images.githubusercontent.com/46324452/149438775-20360279-10ef-4963-8d6a-188e934bb0c2.png)
+
+## Uses for HERE data:
+We use HERE data to:
+- Calculate travel time for aggregated time periods of at least three weeks for any two locations in the City (which can be used to measure the effects of infrastructure improvements like bike lanes);
+- Examine changes to the Travel Time Index over time (to see the effects of road closures or Provincially mandated lock downs); and,
+- Determine the 85th percentile speed (to inform speed limit or enforcement initiatives).
+
+## Types of HERE data products
+HERE Technologies provides us with a few different datasets, such as:
+- traffic analytics tables containing probe data for all Toronto streets
+- traffic patterns tables containing aggregated speed information for all Toronto streets
+- street attribute tables showing speed limits, street names one way info (and more!) for Toronto's streets
+- many GIS layers of streets, intersections, points of interest, trails, rail corridors and water features.
+
+## HERE Data: dates and time periods
+As stated above, the traffic analytics table (here.ta) is updated daily. 
+
+Other products are generally updated quarterly, though this schedule has been somewhat disrupted due to the pandemic.
+
+There are also aggregated tables or views specific to certain modes (like trucks or cyclists) or time periods (like weekends or night-time).
+
+We have HERE data going back to 2012, but there sure weren't as many people walking around with "probes" in their pockets back then! Therefore, 2014 is generally seen as the first year for which travel times can be reasonably relied upon. 
+
+Travel times are calculated for time periods that are at least three weeks or longer. Shorter time periods will not generate enough accurate probe data. Longer time periods and roads with higher traffic volumes have more probe data, and therefore more accurate results, than short time periods on roads with light traffic. More probe data increases accuracy. There are some road segments in the City (cul-de-sacs and other lightly travelled roads) that have very few observations.
+
+# HERE data at a glance
+
+The Question       | The Answer     |
+:----------------- | :---------------
+What is the HERE dataset used for? | To calculate travel times and monitor congestion (mostly)
+Where is the HERE dataset from? | HERE Technologies, via an agreement with the Ontario Ministry of Transportation (MTO)
+Is it available on Open Data? | No
+What area does the HERE dataset represent? | All of Toronto; the co-ordinate system is EPSG: 4326
+Where is it stored? | On an internal postgres database called bigdata; in several schema (here, here_analysis, here_eval and here_gis)
```

Comment:
I feel like the answer to this question for just about every dataset in this repo is going to be "On an internal postgres database called bigdata".

Reply:
Yep